### ISSUE TITLE
doesn't work

### ISSUE THREAD

**Issue by oslnx** [julian-klode/dns66#317]
this is an another crap that we waste time to install and test.

![photo_2018-12-12_08-44-10](https://user-images.githubusercontent.com/1692710/49864866-9ba11580-fdea-11e8-8743-70806d501c24.jpg)

**Comment by patrickdrd** [comment]
no, it's not crap, I was using it for a long time with great results,
you should know that ad displaying depends on a lot of things:
first of all, an app could have embedded apps, like instagram, in that case,
there is nothing any app can do,
secondly, an app can use ip addresses to access ad servers,
nothing you can do either,
and last but not least, it depends on the list(s) you're using,
you should search and add decents lists and then blame the app

**Comment by dragonpt** [comment]
Man really?
You could be polite.

This does works of course, there's many ways adds can reach your device

If your clueless how things work, don't come here and drop the c* bomb

**Comment by saraneth0o** [comment]
Works for me for the one thing I wanted it to and that is blocking ads on the youtube app.
